### PR TITLE
[ES] Clean up `Slider` and `Beatmap`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Hit_object/Slider/es.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
 ---
-no_native_review: true
```
Comment (TicClick) [inline wiki/Gameplay/Hit_object/Slider/es.md:2]: this was set on your own translation, I think it should take another user's review to get it removed

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Hit_object/Slider/es.md [hunk 2]
```diff
@@ -14,17 +14,17 @@ tags:
...
+Un *slider* es un [objeto](/wiki/Gameplay/Hit_object) en [osu!](/wiki/Game_mode/osu!), que consta de un sliderhead, un sliderbody y un slidertail. Una vez que el círculo de aproximación alcanza el borde del sliderhead, como con los [círclulos](/wiki/Gameplay/Hit_object/Hit_circle), el jugador debe hacer clic/tocar el comienzo del slider y luego, mantener presionado el botón, seguir una bola en movimiento (llamada bola del slider) a lo largo de la pista hasta que se alcance el slidertail. Si hay una flecha inversa, el jugador debe seguir la bola del slider hacia atrás por el mismo camino. Los sliders con flechas invertidas se denominan [sliders inversos](/wiki/Gameplay/Hit_object/Slider/Reverse_slider). La velocidad de un slider se define por su [velocidad del slider](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) y el multiplicador de velocidad del slider de la sección de tiempo actual.
```
Comment (MChecaH) [inline wiki/Gameplay/Hit_object/Slider/es.md:17]:
```suggestion
Un *slider* es un [objeto](/wiki/Gameplay/Hit_object) en [osu!](/wiki/Game_mode/osu!), que consta de un sliderhead, un sliderbody y un slidertail. Una vez que el círculo de aproximación alcanza el borde del sliderhead, como con los [círclulos](/wiki/Gameplay/Hit_object/Hit_circle), el jugador debe hacer clic/tocar el comienzo del slider y luego, manteniendo presionado el botón, seguir una bola en movimiento (llamada bola del slider) a lo largo de la pista hasta que se alcance el slidertail. Si hay una flecha inversa, el jugador debe seguir la bola del slider hacia atrás por el mismo camino. Los sliders con flechas invertidas se denominan [sliders inversos](/wiki/Gameplay/Hit_object/Slider/Reverse_slider). La velocidad de un slider se define por su [velocidad del slider](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) y el multiplicador de velocidad del slider de la sección de tiempo actual.
```

It's important keeping the gerund as you're following the ball *while* you're holding the button. Not just pressing it and releasing it.

File: wiki/Gameplay/Hit_object/Slider/es.md [hunk 3]
```diff
@@ -34,4 +34,4 @@ Para cambiar el volumen de un en la mitad del slider, el sampleset actual debe c
 
 ## Técnicas de slider
 
-Los sliders a menudo se asignan a sonidos largos o sostenidos en la música, pero se pueden usar en cualquier lugar siempre que haya un ritmo que los apoye. Los sliders se utilizan para hacer que la lectura sea intuitiva cuando las canciones tienen cambios de ritmo, ya que los slidertail dan retroalimentación pero se reproducen de forma pasiva. os sliders también se utilizan para simplificar el ritmo en las dificultades más fáciles.
+Los sliders a menudo se asignan a sonidos largos o sostenidos en la música, pero se pueden usar en cualquier lugar siempre que haya un ritmo que los apoye. Los sliders se utilizan para hacer que la lectura sea intuitiva cuando las canciones tienen cambios de ritmo, ya que los slidertail dan feedback pero se reproducen de forma pasiva. os sliders también se utilizan para simplificar el ritmo en las dificultades más fáciles.
```
Comment (MChecaH) [inline wiki/Gameplay/Hit_object/Slider/es.md:37]:
```suggestion
Los sliders a menudo se asignan a sonidos largos o sostenidos en la música, pero se pueden usar en cualquier lugar siempre que haya un ritmo que los apoye. Los sliders se utilizan para hacer que la lectura sea intuitiva cuando las canciones tienen cambios de ritmo, ya que los slidertail dan feedback pero se reproducen de forma pasiva. Los sliders también se utilizan para simplificar el ritmo en las dificultades más fáciles.
```

**L**

**Review by MChecaH — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Hit_object/Slider/es.md (hunk 3 above)
Comment (agatemosu) [reply wiki/Gameplay/Hit_object/Slider/es.md:37]: L tu :rage:

**Review by TicClick — APPROVED** [review]

**Pull request merged**